packages/trip-form/src/SettingsSelectorPanel.story.tsx

```diff
@@ -0,0 +1,156 @@
+import flatten from "flat";
+import { ClassicModeIcon } from "@opentripplanner/icons";
+import { action } from "@storybook/addon-actions";
+import React, { Component, ReactElement } from "react";
+import { IntlProvider } from "react-intl";
+import SettingsSelectorPanel from "./SettingsSelectorPanel";
+
+import commonCompanies from "./__mocks__/companies";
+import commonModesEmpty from "./__mocks__/modes-empty";
+import trimet from "./__mocks__/trimet-styled";
+
+import englishMessages from "../i18n/en-US.yml";
+import frenchMessages from "../i18n/fr.yml";
+
+// Customize a few strings from query-params to demonstrate French locale.
+// Normally, a translator would go through the entire query-params file.
```

Comment:
Are we decided on this approach? Would it not be possible for the query-params file to contain internationalization keys that then get fed through `formatMessage` here? That feels cleaner than creating many query-params files (in case the query params ever update)